utils/check_repo.py

```diff
@@ -185,6 +185,7 @@
     "TFWav2Vec2ForCTC",
     "TFHubertForCTC",
     "MaskFormerForInstanceSegmentation",
+    "SplinterForPreTraining",
```

Comment:
Why not add the model to the `AutoModelForPreTraining` class?

Reply:
added